packages/ebayui-core-react/src/ebay-page-notice/__tests__/index.stories.tsx

```diff
@@ -50,7 +50,7 @@ export const InformationMessage = () => (
 
 export const AttentionMessage = () => (
     <>
-        <EbayPageNotice status="attention" aria-label="Attention">
+        <EbayPageNotice status="attention" aria-label="Attention" className="custom-class">
```

Comment:
Can you add this instead on the `index.spec.tsx` file as a separate test case that is more specific to `custom-class` instead of updating the existing one